### PR TITLE
Update dependencies to Vue CLI 3.

### PR DESCRIPTION
**Pull request by Deeadline**
Solves #4

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,48 +1,17 @@
-// http://eslint.org/docs/user-guide/configuring
-
 module.exports = {
```
Comment (magnobiet) [inline .eslintrc.js]: Use this rules, please.

```javascript
module.exports = {
	root: true,
	env: {
		node: true
	},
	extends: [
		'eslint:recommended',
		'plugin:vue/essential'
	],
	rules: {
		'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
		'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
		'linebreak-style': ['error', 'unix'],
		'quotes': ['error', 'single'],
		'camelcase': 'error',
		'semi': ['error', 'always'],
		'vue/attribute-hyphenation': ['error', 'always'],
		'vue/html-end-tags': 'error',
		'vue/html-indent': ['error', 'tab'],
		'vue/html-self-closing': 'error',
		'vue/require-default-prop': 'error',
		'vue/require-prop-types': 'error',
		'vue/attributes-order': 'error',
		'vue/html-quotes': ['error', 'double'],
		'vue/order-in-components': 'error'
	},
	parserOptions: {
		parser: 'babel-eslint'
	}
};
```

File: .gitignore [hunk 2]
```diff
@@ -1,138 +1,24 @@
-# Created by https://www.gitignore.io/api/sass,node,linux,macos,windows
```
Comment (magnobiet) [inline .gitignore]: Preserve this config.

**Comment by Deeadline** [comment]
Hello, I think, I solved all problems ;)

**Comment by magnobiet** [comment]
We have some small problems yet, can you solve it?

```bash
$ vue-cli-service lint
error: Type of the default value for 'character' prop must be a function (vue/require-valid-default-prop) at src/components/marvel/CharacterCard.vue:42:5:
  40 |                  character: {
  41 |                          type: Object,
> 42 |                          default: {}
     |                          ^
  43 |                  }
  44 |          }
  45 |  };


error: Type of the default value for 'comics' prop must be a function (vue/require-valid-default-prop) at src/components/marvel/CharacterComicsCard.vue:34:5:
  32 |                  comics: {
  33 |                          type: Object,
> 34 |                          default: {}
     |                          ^
  35 |                  }
  36 |          }
  37 |  };


error: Type of the default value for 'stories' prop must be a function (vue/require-valid-default-prop) at src/components/marvel/CharacterStoriesCard.vue:34:5:
  32 |                  stories: {
  33 |                          type: Object,
> 34 |                          default: {}
     |                          ^
  35 |                  }
  36 |          }
  37 |  };


error: The "components" property should be above the "data" property on line 69 (vue/order-in-components) at src/components/pages/PageMarvelCharacter.vue:77:3:
  75 |                  ])
  76 |          },
> 77 |          components: {
     |          ^
  78 |                  MarvelCharacterComicsCard,
  79 |                  MarvelCharacterStoriesCard
  80 |          },


error: The "created" property should be above the "methods" property on line 81 (vue/order-in-components) at src/components/pages/PageMarvelCharacter.vue:86:3:
  84 |                  ])
  85 |          },
> 86 |          created() {
     |          ^
  87 |
  88 |                  this.FETCH_CHARACTER({
  89 |                          id: this.$route.params.id


error: The "components" property should be above the "data" property on line 57 (vue/order-in-components) at src/components/pages/PageMarvelCharacters.vue:72:3:
  70 |                  }
  71 |          },
> 72 |          components: {
     |          ^
  73 |                  MarvelCharacterCard,
  74 |                  MarvelCharacterSearch
  75 |          },


error: The "created" property should be above the "methods" property on line 76 (vue/order-in-components) at src/components/pages/PageMarvelCharacters.vue:98:3:
   96 |                         }
   97 |                 },
>  98 |                 created() {
      |                 ^
   99 |
  100 |                         this.loading(true);
  101 |


7 errors found.

 ERROR  Error: Process exited with code 1

Error: Process exited with code 1
    at exports.exit (marvel-app/node_modules/@vue/cli-shared-utils/lib/exit.js:7:11)
    at lint (marvel-app/node_modules/@vue/cli-plugin-eslint/lint.js:86:5)
    at api.registerCommand.args (marvel-app/node_modules/@vue/cli-plugin-eslint/index.js:57:22)
    at Service.run (marvel-app/node_modules/@vue/cli-service/lib/Service.js:203:12)
    at Object.<anonymous> (marvel-app/node_modules/@vue/cli-service/bin/vue-cli-service.js:36:9)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)

Total task duration: 1.12s

```

**Comment by Deeadline** [comment]
Done. Green screen :D

**Pull request merged**